projects/packages/videopress/src/client/block-editor/blocks/video/components/videopress-uploader/uploader-editor.js

```diff
@@ -75,7 +75,7 @@ const PosterActions = ( { hasPoster, onSelectPoster, onRemovePoster } ) => {
 	return (
 		<span className="uploading-editor__scrubber-help">
 			{ __(
-				'This is how the video will look. Use the slider to choose a poster image.',
+				'This is how the video will look. Use the slider to choose a poster image or adjust it in the Block Settings Sidebar under Poster.',
```

Comment:
afaik the section is named "poster and preview". Should we adjust wording based on that?

Reply:
checked and I see it's called "Poster and preview" in Calypso but simply "Poster" in WP Admin. I vote to change the direction to "Use the slider to choose a poster image or change it from the block settings" which is agnostic while still communicating that the poster image can be changed from the block settings

Reply:
Yeah, I like that option